spec/requests/api/v0/post_tokens_spec.rb

```diff
@@ -2,35 +2,27 @@
 
 describe 'POST /api/v0/tokens' do
   it 'returns a 401 and no token when auth fails' do
-    VCR.use_cassette('auth_failure') do
```

Comment:
we can remove this cassette if we are no longer using it. do you think we *should* be using it somewhere if not here? I am not sure what the best line is between mocking CDX and using VCR